src/nile/cli.py

```diff
@@ -115,15 +118,17 @@ def setup(signer, network):
 @click.argument("params", nargs=-1)
 @click.option("--max_fee", nargs=1)
 @network_option
-def send(signer, contract_name, method, params, network, max_fee=None):
+def send(signer, address_or_alias, method, params, network, max_fee=None):
```

Comment:
Good call! I believe we also have to update the `click.argument` to `address_or_alias`

Reply:
Yes, I missed that. Updated already :)